content/numpy/concepts/math-methods/math-methods.md

```diff
@@ -0,0 +1,71 @@
+
+
+# Math Methods
+  - Numerical Python abreviated as numPy, used for numerical computing.
+  - It provides a wide range of mathematical functions and operations on scalar values, arrays and matrices.
+  - Acts as a fundamental tool for scientific computing, data analysis, and machine learning.
+  - NumPy's math methods facilitate complex mathematical computations efficiently, making it indispensable in various domains including scientific research, engineering, finance, and data analysis.
+  - Understanding these methods empowers practitioners to leverage the full potential of NumPy for their numerical computing tasks.
+## Syntax
+
+ numpy. <math_method>
+
+## Example
+![image](https://github.com/gitsoft145/docs/assets/78934620/7f0cf44a-f91d-4ef2-95e5-7cb065387980)
```

Comment:
```suggestion
Instead of using an image, add the code block here, as follows:

```pseudo
import numpy as np

//add two scalar numbers using numpy.add() method
result = np.add(5, 3)

print("Result of addition:", result)
```

```